shared/teams/team/container.js

```diff
@@ -178,9 +172,9 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
   let youImplicitAdmin = false
   let youAreMember = false
   if (you) {
-    youExplicitAdmin = isExplicitAdmin(stateProps._memberInfo, you)
+    youExplicitAdmin = Constants.isOwner(stateProps.yourRole) || Constants.isAdmin(stateProps.yourRole)
     youImplicitAdmin = stateProps._implicitAdminUsernames.has(you)
-    youAreMember = stateProps._memberInfo.some(member => member.username === you)
+    youAreMember = stateProps.yourRole !== '' && stateProps.yourRole !== 'none'
```

Comment:
kind of a smell to have to check for two different empty values, instead of `null` or `!!`

Reply:
@akalin-keybase Yeah I agree, I changed `getRole` to return `null` if there's no team entry, but core tells us `'none'` if it's a subteam you're an implicit admin of so I think we still need the second check